test/payment/SplitPayment.test.js

```diff
@@ -52,7 +52,7 @@ contract('SplitPayment', function ([_, owner, payee1, payee2, payee3, nonpayee1,
 
     it('should have payees', async function () {
       this.payees.forEach(async (payee, index) => {
```

Comment:
We need to await each of the promises from inside the `it` block: could you replace this `forEach` with an `await Promise.all(this.payees.map(async (payee, index) => {`?

Reply:
Yes…